### PR TITLE
Added button to delete files from the Explore page

### PR DESCRIPTION
**Pull request by shuruheel**

### CODE REVIEW

File: backend/api.py [hunk 1]
```diff
@@ -166,7 +166,12 @@ async def explore_endpoint():
 @app.delete("/explore/{file_name}")
 async def delete_endpoint(file_name: str):
     response = supabase.table("documents").delete().match({"metadata->>file_name": file_name}).execute()
-    return {"message": f"{file_name} has been deleted."}
```
Comment (Shaunwei) [inline backend/api.py:169]: revert change here?

File: backend/api.py
Comment (Shaunwei) [inline backend/api.py:1]: I don't think u need to update anything in the backend :D

Comment (StanGirard) [reply backend/api.py:1]: Agreed ! However because this is Shan's first i'll accept :)

**Comment by StanGirard** [comment]
@shuruheel Can you please update your PR with the update on the main branch ?

**Pull request closed without merging**